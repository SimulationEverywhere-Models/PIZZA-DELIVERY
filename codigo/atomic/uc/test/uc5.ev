00:00:01:000 abrir_uc 1
00:00:02:000 atiende_uc 2
00:00:08:000 atiende_uc 3
00:00:15:000 atiende_uc 4
00:00:20:000 lleno_p 1
00:00:35:000 recibe_ent_uc 3
00:00:40:000 lista_p_uc 1
00:00:45:000 moto_ocupada_uc 1
00:01:05:000 atiende_uc 5
00:01:10:000 moto_libre_uc 1
00:01:15:000 recibe_ent_uc 2
00:01:20:000 atiende_uc 6
00:01:30:000 atiende_uc 7
00:01:55:000 lleno_ent_uc 1
00:02:10:000 recibe_ent_uc 4
00:02:15:000 moto_ocupada_uc 1
00:02:20:000 lista_ent_uc 1
00:02:30:000 moto_libre_uc 1
00:02:45:000 recibe_ent_uc 6
00:03:00:000 cerrar_uc 1
00:03:15:000 recibe_ent_uc 7
00:03:30:000 recibe_ent_uc 5