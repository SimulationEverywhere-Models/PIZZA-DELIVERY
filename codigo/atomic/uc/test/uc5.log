Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para uc(02)
Mensaje D / 00:00:00:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / abrir_uc /      1.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / abrir_uc /      1.00000 para uc(02)
Mensaje D / 00:00:01:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:01:000 / Root(00) para top(01)
Mensaje * / 00:00:01:000 / top(01) para uc(02)
Mensaje Y / 00:00:01:000 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:00:01:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:01:000 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:00:01:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / atiende_uc /      2.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / atiende_uc /      2.00000 para uc(02)
Mensaje D / 00:00:02:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:02:000 / Root(00) para top(01)
Mensaje * / 00:00:02:000 / top(01) para uc(02)
Mensaje Y / 00:00:02:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:02:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:02:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:02:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:04:000 / Root(00) para top(01)
Mensaje * / 00:00:04:000 / top(01) para uc(02)
Mensaje Y / 00:00:04:000 / uc(02) / encolar_p /      2.00000 para top(01)
Mensaje D / 00:00:04:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:04:000 / top(01) / encolar_p /      2.00000 para Root(00)
Mensaje D / 00:00:04:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:04:000 / Root(00) para top(01)
Mensaje * / 00:00:04:000 / top(01) para uc(02)
Mensaje Y / 00:00:04:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:04:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:04:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:04:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / atiende_uc /      3.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / atiende_uc /      3.00000 para uc(02)
Mensaje D / 00:00:08:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para uc(02)
Mensaje Y / 00:00:08:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:08:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:08:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:08:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para uc(02)
Mensaje Y / 00:00:10:000 / uc(02) / encolar_p /      3.00000 para top(01)
Mensaje D / 00:00:10:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:10:000 / top(01) / encolar_p /      3.00000 para Root(00)
Mensaje D / 00:00:10:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para uc(02)
Mensaje Y / 00:00:10:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:10:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:10:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:10:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:15:000 / Root(00) / atiende_uc /      4.00000 para top(01)
Mensaje X / 00:00:15:000 / top(01) / atiende_uc /      4.00000 para uc(02)
Mensaje D / 00:00:15:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:15:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:15:000 / Root(00) para top(01)
Mensaje * / 00:00:15:000 / top(01) para uc(02)
Mensaje Y / 00:00:15:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:15:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:15:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:15:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:17:000 / Root(00) para top(01)
Mensaje * / 00:00:17:000 / top(01) para uc(02)
Mensaje Y / 00:00:17:000 / uc(02) / encolar_p /      4.00000 para top(01)
Mensaje D / 00:00:17:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:17:000 / top(01) / encolar_p /      4.00000 para Root(00)
Mensaje D / 00:00:17:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:17:000 / Root(00) para top(01)
Mensaje * / 00:00:17:000 / top(01) para uc(02)
Mensaje Y / 00:00:17:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:17:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:17:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:17:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:20:000 / Root(00) / lleno_p /      1.00000 para top(01)
Mensaje X / 00:00:20:000 / top(01) / lleno_p /      1.00000 para uc(02)
Mensaje D / 00:00:20:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:20:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:20:000 / Root(00) para top(01)
Mensaje * / 00:00:20:000 / top(01) para uc(02)
Mensaje Y / 00:00:20:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:00:20:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:20:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:20:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:35:000 / Root(00) / recibe_ent_uc /      3.00000 para top(01)
Mensaje X / 00:00:35:000 / top(01) / recibe_ent_uc /      3.00000 para uc(02)
Mensaje D / 00:00:35:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:35:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:35:000 / Root(00) para top(01)
Mensaje * / 00:00:35:000 / top(01) para uc(02)
Mensaje Y / 00:00:35:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:35:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:35:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:35:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:35:030 / Root(00) para top(01)
Mensaje * / 00:00:35:030 / top(01) para uc(02)
Mensaje Y / 00:00:35:030 / uc(02) / dist_out_uc /      3.00000 para top(01)
Mensaje D / 00:00:35:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:35:030 / top(01) / dist_out_uc /      3.00000 para Root(00)
Mensaje D / 00:00:35:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:35:030 / Root(00) para top(01)
Mensaje * / 00:00:35:030 / top(01) para uc(02)
Mensaje Y / 00:00:35:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:35:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:35:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:35:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:40:000 / Root(00) / lista_p_uc /      1.00000 para top(01)
Mensaje X / 00:00:40:000 / top(01) / lista_p_uc /      1.00000 para uc(02)
Mensaje D / 00:00:40:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:40:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:40:000 / Root(00) para top(01)
Mensaje * / 00:00:40:000 / top(01) para uc(02)
Mensaje Y / 00:00:40:000 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:00:40:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:40:000 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:00:40:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:45:000 / Root(00) / moto_ocupada_uc /      1.00000 para top(01)
Mensaje X / 00:00:45:000 / top(01) / moto_ocupada_uc /      1.00000 para uc(02)
Mensaje D / 00:00:45:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:45:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:45:000 / Root(00) para top(01)
Mensaje * / 00:00:45:000 / top(01) para uc(02)
Mensaje D / 00:00:45:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:45:000 / top(01) / ... para Root(00)
Mensaje X / 00:01:05:000 / Root(00) / atiende_uc /      5.00000 para top(01)
Mensaje X / 00:01:05:000 / top(01) / atiende_uc /      5.00000 para uc(02)
Mensaje D / 00:01:05:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:05:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:05:000 / Root(00) para top(01)
Mensaje * / 00:01:05:000 / top(01) para uc(02)
Mensaje Y / 00:01:05:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:01:05:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:01:05:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:01:05:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:01:07:000 / Root(00) para top(01)
Mensaje * / 00:01:07:000 / top(01) para uc(02)
Mensaje Y / 00:01:07:000 / uc(02) / encolar_p /      5.00000 para top(01)
Mensaje D / 00:01:07:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:01:07:000 / top(01) / encolar_p /      5.00000 para Root(00)
Mensaje D / 00:01:07:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:07:000 / Root(00) para top(01)
Mensaje * / 00:01:07:000 / top(01) para uc(02)
Mensaje Y / 00:01:07:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:01:07:000 / uc(02) / ... para top(01)
Mensaje Y / 00:01:07:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:01:07:000 / top(01) / ... para Root(00)
Mensaje X / 00:01:10:000 / Root(00) / moto_libre_uc /      1.00000 para top(01)
Mensaje X / 00:01:10:000 / top(01) / moto_libre_uc /      1.00000 para uc(02)
Mensaje D / 00:01:10:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:10:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:10:000 / Root(00) para top(01)
Mensaje * / 00:01:10:000 / top(01) para uc(02)
Mensaje D / 00:01:10:000 / uc(02) / ... para top(01)
Mensaje D / 00:01:10:000 / top(01) / ... para Root(00)
Mensaje X / 00:01:15:000 / Root(00) / recibe_ent_uc /      2.00000 para top(01)
Mensaje X / 00:01:15:000 / top(01) / recibe_ent_uc /      2.00000 para uc(02)
Mensaje D / 00:01:15:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:15:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:15:000 / Root(00) para top(01)
Mensaje * / 00:01:15:000 / top(01) para uc(02)
Mensaje Y / 00:01:15:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:01:15:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:01:15:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:01:15:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:01:15:030 / Root(00) para top(01)
Mensaje * / 00:01:15:030 / top(01) para uc(02)
Mensaje Y / 00:01:15:030 / uc(02) / dist_out_uc /      2.00000 para top(01)
Mensaje D / 00:01:15:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:01:15:030 / top(01) / dist_out_uc /      2.00000 para Root(00)
Mensaje D / 00:01:15:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:15:030 / Root(00) para top(01)
Mensaje * / 00:01:15:030 / top(01) para uc(02)
Mensaje Y / 00:01:15:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:01:15:030 / uc(02) / ... para top(01)
Mensaje Y / 00:01:15:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:01:15:030 / top(01) / ... para Root(00)
Mensaje X / 00:01:20:000 / Root(00) / atiende_uc /      6.00000 para top(01)
Mensaje X / 00:01:20:000 / top(01) / atiende_uc /      6.00000 para uc(02)
Mensaje D / 00:01:20:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:20:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:20:000 / Root(00) para top(01)
Mensaje * / 00:01:20:000 / top(01) para uc(02)
Mensaje Y / 00:01:20:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:01:20:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:01:20:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:01:20:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:01:22:000 / Root(00) para top(01)
Mensaje * / 00:01:22:000 / top(01) para uc(02)
Mensaje Y / 00:01:22:000 / uc(02) / encolar_p /      6.00000 para top(01)
Mensaje D / 00:01:22:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:01:22:000 / top(01) / encolar_p /      6.00000 para Root(00)
Mensaje D / 00:01:22:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:22:000 / Root(00) para top(01)
Mensaje * / 00:01:22:000 / top(01) para uc(02)
Mensaje Y / 00:01:22:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:01:22:000 / uc(02) / ... para top(01)
Mensaje Y / 00:01:22:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:01:22:000 / top(01) / ... para Root(00)
Mensaje X / 00:01:30:000 / Root(00) / atiende_uc /      7.00000 para top(01)
Mensaje X / 00:01:30:000 / top(01) / atiende_uc /      7.00000 para uc(02)
Mensaje D / 00:01:30:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:30:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:30:000 / Root(00) para top(01)
Mensaje * / 00:01:30:000 / top(01) para uc(02)
Mensaje Y / 00:01:30:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:01:30:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:01:30:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:01:30:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:01:32:000 / Root(00) para top(01)
Mensaje * / 00:01:32:000 / top(01) para uc(02)
Mensaje Y / 00:01:32:000 / uc(02) / encolar_p /      7.00000 para top(01)
Mensaje D / 00:01:32:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:01:32:000 / top(01) / encolar_p /      7.00000 para Root(00)
Mensaje D / 00:01:32:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:32:000 / Root(00) para top(01)
Mensaje * / 00:01:32:000 / top(01) para uc(02)
Mensaje Y / 00:01:32:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:01:32:000 / uc(02) / ... para top(01)
Mensaje Y / 00:01:32:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:01:32:000 / top(01) / ... para Root(00)
Mensaje X / 00:01:55:000 / Root(00) / lleno_ent_uc /      1.00000 para top(01)
Mensaje X / 00:01:55:000 / top(01) / lleno_ent_uc /      1.00000 para uc(02)
Mensaje D / 00:01:55:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:01:55:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:01:55:000 / Root(00) para top(01)
Mensaje * / 00:01:55:000 / top(01) para uc(02)
Mensaje Y / 00:01:55:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:01:55:000 / uc(02) / ... para top(01)
Mensaje Y / 00:01:55:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:01:55:000 / top(01) / ... para Root(00)
Mensaje X / 00:02:10:000 / Root(00) / recibe_ent_uc /      4.00000 para top(01)
Mensaje X / 00:02:10:000 / top(01) / recibe_ent_uc /      4.00000 para uc(02)
Mensaje D / 00:02:10:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:02:10:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:10:000 / Root(00) para top(01)
Mensaje * / 00:02:10:000 / top(01) para uc(02)
Mensaje Y / 00:02:10:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:02:10:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:02:10:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:02:10:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:02:10:030 / Root(00) para top(01)
Mensaje * / 00:02:10:030 / top(01) para uc(02)
Mensaje Y / 00:02:10:030 / uc(02) / dist_out_uc /      4.00000 para top(01)
Mensaje D / 00:02:10:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:02:10:030 / top(01) / dist_out_uc /      4.00000 para Root(00)
Mensaje D / 00:02:10:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:10:030 / Root(00) para top(01)
Mensaje * / 00:02:10:030 / top(01) para uc(02)
Mensaje Y / 00:02:10:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:02:10:030 / uc(02) / ... para top(01)
Mensaje Y / 00:02:10:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:02:10:030 / top(01) / ... para Root(00)
Mensaje X / 00:02:15:000 / Root(00) / moto_ocupada_uc /      1.00000 para top(01)
Mensaje X / 00:02:15:000 / top(01) / moto_ocupada_uc /      1.00000 para uc(02)
Mensaje D / 00:02:15:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:02:15:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:15:000 / Root(00) para top(01)
Mensaje * / 00:02:15:000 / top(01) para uc(02)
Mensaje D / 00:02:15:000 / uc(02) / ... para top(01)
Mensaje D / 00:02:15:000 / top(01) / ... para Root(00)
Mensaje X / 00:02:20:000 / Root(00) / lista_ent_uc /      1.00000 para top(01)
Mensaje X / 00:02:20:000 / top(01) / lista_ent_uc /      1.00000 para uc(02)
Mensaje D / 00:02:20:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:02:20:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:20:000 / Root(00) para top(01)
Mensaje * / 00:02:20:000 / top(01) para uc(02)
Mensaje Y / 00:02:20:000 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:02:20:000 / uc(02) / ... para top(01)
Mensaje Y / 00:02:20:000 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:02:20:000 / top(01) / ... para Root(00)
Mensaje X / 00:02:30:000 / Root(00) / moto_libre_uc /      1.00000 para top(01)
Mensaje X / 00:02:30:000 / top(01) / moto_libre_uc /      1.00000 para uc(02)
Mensaje D / 00:02:30:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:02:30:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:30:000 / Root(00) para top(01)
Mensaje * / 00:02:30:000 / top(01) para uc(02)
Mensaje D / 00:02:30:000 / uc(02) / ... para top(01)
Mensaje D / 00:02:30:000 / top(01) / ... para Root(00)
Mensaje X / 00:02:45:000 / Root(00) / recibe_ent_uc /      6.00000 para top(01)
Mensaje X / 00:02:45:000 / top(01) / recibe_ent_uc /      6.00000 para uc(02)
Mensaje D / 00:02:45:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:02:45:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:45:000 / Root(00) para top(01)
Mensaje * / 00:02:45:000 / top(01) para uc(02)
Mensaje Y / 00:02:45:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:02:45:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:02:45:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:02:45:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:02:45:030 / Root(00) para top(01)
Mensaje * / 00:02:45:030 / top(01) para uc(02)
Mensaje Y / 00:02:45:030 / uc(02) / dist_out_uc /      6.00000 para top(01)
Mensaje D / 00:02:45:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:02:45:030 / top(01) / dist_out_uc /      6.00000 para Root(00)
Mensaje D / 00:02:45:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:02:45:030 / Root(00) para top(01)
Mensaje * / 00:02:45:030 / top(01) para uc(02)
Mensaje Y / 00:02:45:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:02:45:030 / uc(02) / ... para top(01)
Mensaje Y / 00:02:45:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:02:45:030 / top(01) / ... para Root(00)
Mensaje X / 00:03:00:000 / Root(00) / cerrar_uc /      1.00000 para top(01)
Mensaje X / 00:03:00:000 / top(01) / cerrar_uc /      1.00000 para uc(02)
Mensaje D / 00:03:00:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:03:00:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:03:00:000 / Root(00) para top(01)
Mensaje * / 00:03:00:000 / top(01) para uc(02)
Mensaje Y / 00:03:00:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:03:00:000 / uc(02) / ... para top(01)
Mensaje Y / 00:03:00:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:03:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:03:15:000 / Root(00) / recibe_ent_uc /      7.00000 para top(01)
Mensaje X / 00:03:15:000 / top(01) / recibe_ent_uc /      7.00000 para uc(02)
Mensaje D / 00:03:15:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:03:15:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:03:15:000 / Root(00) para top(01)
Mensaje * / 00:03:15:000 / top(01) para uc(02)
Mensaje Y / 00:03:15:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:03:15:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:03:15:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:03:15:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:03:15:030 / Root(00) para top(01)
Mensaje * / 00:03:15:030 / top(01) para uc(02)
Mensaje Y / 00:03:15:030 / uc(02) / dist_out_uc /      7.00000 para top(01)
Mensaje D / 00:03:15:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:03:15:030 / top(01) / dist_out_uc /      7.00000 para Root(00)
Mensaje D / 00:03:15:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:03:15:030 / Root(00) para top(01)
Mensaje * / 00:03:15:030 / top(01) para uc(02)
Mensaje Y / 00:03:15:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:03:15:030 / uc(02) / ... para top(01)
Mensaje Y / 00:03:15:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:03:15:030 / top(01) / ... para Root(00)
Mensaje X / 00:03:30:000 / Root(00) / recibe_ent_uc /      5.00000 para top(01)
Mensaje X / 00:03:30:000 / top(01) / recibe_ent_uc /      5.00000 para uc(02)
Mensaje D / 00:03:30:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:03:30:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:03:30:000 / Root(00) para top(01)
Mensaje * / 00:03:30:000 / top(01) para uc(02)
Mensaje Y / 00:03:30:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:03:30:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:03:30:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:03:30:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:03:30:030 / Root(00) para top(01)
Mensaje * / 00:03:30:030 / top(01) para uc(02)
Mensaje Y / 00:03:30:030 / uc(02) / dist_out_uc /      5.00000 para top(01)
Mensaje D / 00:03:30:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:03:30:030 / top(01) / dist_out_uc /      5.00000 para Root(00)
Mensaje D / 00:03:30:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:03:30:030 / Root(00) para top(01)
Mensaje * / 00:03:30:030 / top(01) para uc(02)
Mensaje Y / 00:03:30:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:03:30:030 / uc(02) / ... para top(01)
Mensaje Y / 00:03:30:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:03:30:030 / top(01) / ... para Root(00)
