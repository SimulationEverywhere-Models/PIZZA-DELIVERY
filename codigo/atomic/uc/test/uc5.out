00:00:01:000 lista_ll_uc 1
00:00:02:000 ocupado_uc 1
00:00:04:000 encolar_p 2
00:00:04:000 libre_uc 1
00:00:08:000 ocupado_uc 1
00:00:10:000 encolar_p 3
00:00:10:000 libre_uc 1
00:00:15:000 ocupado_uc 1
00:00:17:000 encolar_p 4
00:00:17:000 libre_uc 1
00:00:20:000 cerrado_uc 1
00:00:35:000 ocupada_ent_uc 1
00:00:35:030 dist_out_uc 3
00:00:35:030 libre_uc_ent 1
00:00:40:000 lista_ll_uc 1
00:01:05:000 ocupado_uc 1
00:01:07:000 encolar_p 5
00:01:07:000 libre_uc 1
00:01:15:000 ocupada_ent_uc 1
00:01:15:030 dist_out_uc 2
00:01:15:030 libre_uc_ent 1
00:01:20:000 ocupado_uc 1
00:01:22:000 encolar_p 6
00:01:22:000 libre_uc 1
00:01:30:000 ocupado_uc 1
00:01:32:000 encolar_p 7
00:01:32:000 libre_uc 1
00:01:55:000 cerrado_uc 1
00:02:10:000 ocupada_ent_uc 1
00:02:10:030 dist_out_uc 4
00:02:10:030 libre_uc_ent 1
00:02:20:000 lista_ll_uc 1
00:02:45:000 ocupada_ent_uc 1
00:02:45:030 dist_out_uc 6
00:02:45:030 libre_uc_ent 1
00:03:00:000 cerrado_uc 1
00:03:15:000 ocupada_ent_uc 1
00:03:15:030 dist_out_uc 7
00:03:15:030 libre_uc_ent 1
00:03:30:000 ocupada_ent_uc 1
00:03:30:030 dist_out_uc 5
00:03:30:030 libre_uc_ent 1
